test/testUtils.cpp

```diff
@@ -84,15 +96,15 @@ void interpolateWachspressSphericalTest(MPMesh& mpMesh){
     p_MPs->parallel_for(eval, "interpolateWachspressSphericalTest");
 }
 
-void interpolateWachspress3DTest(MPMesh& mpMesh){
+void interpolateWachspress3DTest(MPMesh& mpMesh, const int testMeshOption){
     auto p_mesh = mpMesh.p_mesh;
     auto vtxCoords = p_mesh->getMeshField<polyMPO::MeshF_VtxCoords>();
     auto elm2VtxConn = p_mesh->getElm2VtxConn();
 
     auto p_MPs = mpMesh.p_MPs;
     auto MPsPosition = p_MPs->getPositions();
     auto eval = PS_LAMBDA(const int& elm, const int& mp, const int mask){
-        if (mask) {
+        if (mask && mp == 1) {
```

Comment:
Why only 'mp==1'? I do not recall if this was on purpose or for debugging

Reply:
Right, it was mainly for debugging and I don't think that's needed anymore. I'll remove it